client/question/components/LearnerViewDirective.js

```diff
@@ -906,6 +964,9 @@ tie.directive('learnerView', [function() {
         $scope.resetCode = function() {
           LocalStorageService.clearLocalStorageCode(
             $scope.currentQuestionId, language);
+          EventHandlerService.createCodeResetEvent(
+            SessionIdService.getSessionId());
+          // Do we want to trigger another QuestionStartEvent?
```

Comment:
Yes, I guess? But that's already being done in loadQuestion right?

Reply:
Fair enough.